xmake/modules/target/action/install/cmake_importfiles.lua

```diff
@@ -130,6 +130,16 @@ function _install_cmake_targetfile(target, installdir, filename, opt)
             local value = builtinvars[variable]
             return type(value) == "function" and value() or value
         end)
+        local libfile = path.filename(target:targetfile())
+        local postfix = is_mode("debug") and "DEBUG" or "RELEASE"
+        if target:is_shared() and (_get_libfile(target, installdir) ~= libfile) then
+            -- On DLL platforms, the import library is named differently from the target file
+            content = content:gsub("# IMPORTED_IMPLIB_" .. postfix, "IMPORTED_IMPLIB_" .. postfix)
+            content = content:gsub(
+                "IMPORTED_LOCATION_" .. postfix .. " \"${_IMPORT_PREFIX}/lib/.-\"",
```

Comment:
$ is lua pattern character.

Reply:
If the pattern character is not at the beginning or end of the string, it matches the character literally.